scripts/dev_cli.sh

```diff
@@ -396,6 +396,13 @@ cmd_tests() {
     process_volumes_args
     target="$(uname -m)-unknown-linux-${libc}"
 
+    rustflags="$RUSTFLAGS"
+    target_cc=""
+    if [ "$(uname -m)" = "aarch64" ] && [ "$libc" = "musl" ]; then
+        rustflags="$rustflags -C link-arg=-lgcc -C link_arg=-specs -C link_arg=/usr/lib/aarch64-linux-musl/musl-gcc.specs"
+        target_cc="musl-gcc"
```

Comment:
@liuw Just want to point out that `target_cc` is only set for `aarch64` + `musl`. This is different from previous behavior, where we always set `target_cc="musl-gcc"` as long as we are building with `musl` libc. No sure if this is intended.

Reply:
I noticed that. I _think_ it is for cross-compilation. When we're not cross-compiling, setting it to empty has no effect. That should be the default value anyway.